src/main/java/Group15/View/ExerciseDetailsView.java

```diff
@@ -92,30 +85,54 @@ private static Pane createExerciseInfoPane(Exercise exercise)
         Label difficultyLabel = new Label("Difficulty: " + exercise.difficulty);
         difficultyLabel.setFont(Font.font("Arial", FontWeight.NORMAL, 16));
 
-        infoPane.getChildren().addAll(nameLabel, descriptionLabel, bodyPartLabel, equipmentLabel, difficultyLabel);
+        infoPane.getChildren().addAll(nameLabel, descriptionLabel, typeLabel, bodyPartLabel, equipmentLabel, difficultyLabel);
 
         return infoPane;
     }
 
-    private static Pane createButtonPane()
-    {
+    private static Pane createButtonPane(Exercise exercise) {
         HBox buttonPane = new HBox();
         buttonPane.setAlignment(Pos.CENTER);
         buttonPane.setSpacing(20);
 
-        Button favoriteButton = new Button("Favorite");
-        favoriteButton.setPrefSize(200, 50);
-        favoriteButton.setOnAction(_ ->
-            {
-            //TODO: Add functionality to add exercise to favorites
-            });
+        Button likeButton = new Button("Like");
+        likeButton.setPrefSize(200, 50);
+
+        Button dislikeButton = new Button("Dislike");
+        dislikeButton.setPrefSize(200, 50);
+
+        updateButtonStyles(exercise, likeButton, dislikeButton);
+
+        likeButton.setOnAction(_ -> {
+        ExerciseUtils.likeExercisePressed(exercise);
+        updateButtonStyles(exercise, likeButton, dislikeButton);
+        });
+
+        dislikeButton.setOnAction(_ -> {
+        ExerciseUtils.dislikeExercisePressed(exercise);
+        updateButtonStyles(exercise, likeButton, dislikeButton);
+        });
 
         Button backButton = new Button("Back");
         backButton.setPrefSize(200, 50);
-        backButton.setOnAction(_ -> ViewController.goBack());
+        backButton.setOnAction(_ -> ViewController.setScene(WorkoutView.createScene(new Workout())));
```

Comment:
Are you sure you don't want to call goBack()?

Reply:
I have not made this change, but nice catch :)